### ISSUE TITLE
Video is added on not executed tests

### ISSUE THREAD

**Issue by irapths** [LironEr/cypress-mochawesome-reporter#149]
### Environment

```markdown
- OS: All
- Node: 
- cypress-mochawesome-reporter: 3.5.0
- cypress:
```

### What happened?

The video is added as _Additional Test Context_ on not executed (_SKEPped_) tests.

Moreover, using the new option `videoOnFailOnly` (#141) and set it to `true` does not prevent videos to be added on _SKIPped_ tests.

### Config file

```json
Default settings

OR

videoOnFailOnly: true
```

### Anything else?

The addition of video as _Additional Test Context_ on tests is a great feature.

But I think it should be added only on executed tests (_PASSed_ or _FAILed_) and not on _SKIPped_.
Additionaly the option `videoOnFailOnly: true` should add video only on _FAILed_ tests and prevent the video to be added on all others.

**Comment by joakim-sch** [comment]
Hi @irapths !
Thank you for your feedback. Glad you enjoyed the video feature 😄 🎥 
The logic is just looking at not passed tests when `videoOnFailOnly: true` so there is definitely logic for skipped tests missing... Should be an easy fix tho 😃

**Comment by joakim-sch** [comment]
I have created PR #150 that should fix the issue ✌🏼

**Comment by LironEr** [comment]
Fixed in [v3.5.1](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v3.5.1)
Thanks @joakim-sch and @irapths
